scripts/plugins/storage/sqlite/upgrade/12.sql

```diff
@@ -0,0 +1,36 @@
+DROP TABLE IF EXISTS foglamp.destinations;
+DROP INDEX IF EXISTS foglamp.fki_streams_fk1;
```

Comment:
did we check this? I think its not complaining as we have `IF EXISTS`  you may need to remove `foglamp.`

Reply:
I tested, I retested right now to be sure :  

Downgrade :
``` 
Detected 'sqlite' FogLAMP DB schema change from version [12] to [11], applying Upgrade/Downgrade ...
FogLAMP DB schema has been downgraded to version [11]
Starting FogLAMP v1.3.1......
FogLAMP started.
```

Upgrade : 
```
foglamp@ubuntu:~$ ${FOGLAMP_ROOT}/scripts/foglamp start
Detected 'sqlite' FogLAMP DB schema change from version [11] to [12], applying Upgrade/Downgrade ...
FogLAMP DB schema has been upgraded to version [12]
Starting FogLAMP v1.3.1......
FogLAMP started.
foglamp@ubuntu:~$
```

the table was removed :
```
foglamp@ubuntu:~$ export FOGLAMP_ROOT=/home/foglamp/Development/FogLAMP;\
> cmd_sql='SELECT *  FROM destinations;';\
> sqlite3 -column  -header   -cmd ".timeout 5000"  "file:${FOGLAMP_ROOT}/data/foglamp.db?mode=ro&nolock=1"  "${cmd_sql}"
Error: no such table: destinations
foglamp@ubuntu:~$
```

Reply:
My comment was for ` DROP INDEX IF EXISTS foglamp.fki_streams_fk1; `